doc/holochain_101/src/zome/define_zome.md

```diff
@@ -28,33 +28,33 @@ What are Rust macros? Generally speaking, they are code that will actually gener
 
 In the case of Zome development, it was discovered that much code could be saved from being written, by encapsulating it in a macro.
 
-That is how `define_zome!` came about. It is a Rust macro imported from the HDK which must be used for every Zome (unless you read the source code for it yourself and write something that behaves the same way!)
+That is how `#[zome]` came about. It is a Rust macro imported from the HDK which must be used for every Zome (unless you read the source code for it yourself and write something that behaves the same way!)
 
 The following is technically the most minimalistic Zome that could be implemented. It does nothing, but still conforms to the expectations Holochain has for a Zome.
 
 ```rust
 #[macro_use]
 extern crate hdk;
 
-define_zome! {
-    entries: []
-
-    init: || {
+#[zome]
+mod my_zome {
+    
+    #[init]
+    fn init() -> ZomeApiResult<()> {
         Ok(())
```

Comment:
Important but not urgent:
Showing the simplest useful example of how `init()` would be used (e.g. some dis-allowed states?) would be better than just returning `Ok(())` for all calls in an introductory example like this.